package/CHANGELOG

```diff
@@ -97,6 +97,7 @@ Fixes
   * Contact Analysis class respects PBC (Issue #2368)
 
 Enhancements
+  * Refactored helanal in helix_analysis (Issue #2452, PR #2622)
```

Comment:
this seems a little understated, analysis.helanal is deprecated and analysis.helix_analysis is the new thing right?

Reply:
Yeah, I think I meant "into", is that clearer?

```suggestion
  * Refactored helanal into helix_analysis (Issue #2452, PR #2622)
```